_posts/2020-03-17-postponing-rustfest-nl.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: "Postponing RustFest Netherlands"
+description: "RustFest Netherlands planning for 2020 in times of the Coronavirus"
+socialImageSrc: "/assets/posts/netherlands/postponed.png"
+authors:
+  - djc
+---
+
+For the past months we have been very busy working towards an announcement
+for our 2020 RustFest in the Netherlands, as announced for Q2 of this year.
+Unfortunately it has now become clear that COVID-19 is throwing a wrench in the works --
+even with our planning for an event in early June,
+it is unlikely that the COVID-19 threat will have been contained to a level
+where we can confidently plan getting together with hundreds of awesome Rustaceans
+from around Europe and beyond.
+
+After some intense discussion among our team members, here is our current plan:
+we still want to do a RustFest this year in the Netherlands.
+However, in order to make it accessible to everyone,
+even to those who might be geographically constrained due to containment measures,
+we want to plan both a physical event while also reinventing our event as one that
+can be experienced from home (or, even better, a local meetup group -- COVID-19 willing).
+In order to make this work well, we will need more time to prepare.
+Therefore, we are currently planning for having our RustFest in the fourth quarter,
+with the physical conference happening in Utrecht, the Netherlands.
```

Comment:
Should it be ", in the Netherlands" or just "Netherlands"?

Reply:
Nope, it's fine, it's sometimes explicitly called "the netherlands"

Reply:
I think this is fine as is. It's "Utrecht, the Netherlands" as a geographical location.